frontend/src/app/containers/ExperimentPage/DetailsDescription.js

```diff
@@ -129,20 +129,24 @@ export const LocaleWarning = ({
 };
 
 export const EolBlock = ({ experiment, l10nId }: EolBlockProps) => {
-  const completedDate = formatDate(experiment.completed);
-  const title = `${experiment.title} is ending on ${completedDate}`;
+  const $title = experiment.title;
+  const $completedDate = new FluentDateTime(experiment.completed, {
+    day: "numeric",
+    month: "long",
+    year: "numeric"
+  });
 
   return (
     <Warning
       titleL10nId="eolIntroMessage"
-      titleL10nArgs={JSON.stringify({ title: experiment.title, completedDate })}
-      title={title}
+      titleL10nArgs={{$title, $completedDate}}
+      title={`${experiment.title} is ending on ${experiment.completed}`}
     >
-      <Localized id={l10nId("eolWarning")}>
+      <LocalizedHtml id={l10nId("eolWarning")}>
```

Comment:
None of the current experiments use markup for the EOL warning. It would be safer to use a regular `<Localized>` here.